src/internal/_equals.js

```diff
@@ -1,114 +1,196 @@
+var _containsWith = require('./_containsWith');
 var _arrayFromIterator = require('./_arrayFromIterator');
 var _functionName = require('./_functionName');
 var _has = require('./_has');
 var identical = require('../identical');
 var keys = require('../keys');
 var type = require('../type');
 
+module.exports = (function() {
+  /**
+   * private _uniqContentEquals function
+   * Notes:
+   * 1. this function will return true on [1,2,3] === [1,1,1]
+   * 2. this function do not expect iterables of different size
+   * it works as a shared function for ES6 comparators
```

Comment:
not sure what this line means? "it works as a shared function for ES6 comparators"?